GliaWidgets/SecureConversations/SecureConversations.UnreadMessageLoader.swift

```diff
@@ -0,0 +1,96 @@
+import Dispatch
+import GliaCoreDependency
+
+extension SecureConversations {
+    struct MessagesWithUnreadCount {
+        var messages: [ChatMessage]
+        var unreadCount: Int
+    }
+
+    /// Due to async nature of requests, we need to make sure
+    /// that `unread message count` and `transcript messages`
+    /// are delivered at once. This entity makes it possible.
+    struct MessagesWithUnreadCountLoader {
+        static let unreadCountFallbackTimeoutSeconds: TimeInterval = 3.0
```

Comment:
should it be 3 sec or has it been doubled to 6sec?

Reply:
That is different delay, like in [Androids version](https://github.com/salemove/android-sdk-widgets/blob/29dd51c48ee3ea803745a05df178f0c9d4cf7e5f/widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/GetUnreadMessagesCountWithTimeoutUseCase.kt#L19). 6 sec will be for marking messages read.

Reply:
So messages get marked as read after three seconds, but the message divider gets marked as read after six? That sounds different to what we discussed [in this thread](https://salemove.slack.com/archives/C047N9EAG8Y/p1677076646000439?thread_ts=1675767504.581449&cid=C047N9EAG8Y).

Reply:
This PR is not about marking messages as read. It is about merging two requests into one. Three seconds is the timeout for unread count request.

Reply:
I've updated PR description to make it more clear.